### DISCUSSION TITLE
New SDC User Meeting, on the 25th of February 2022

### DISCUSSION THREAD

**Discussion by violetacln** [sdcTools/UserSupport#217] [User Group]
Dear colleagues,
The 2022 - SDC User Meeting will take place online, on Friday, the 25th of February 2022, 10:00-12:30 C.E.T.
You may register now, by using the form on the [User Support web-page] (https://sdctools.github.io/UserSupport/)!
The detailed agenda and the link for the virtual meeting will be announced in due time.
We are looking forward to hearing from you,
Violeta

**Comment by matthiasgomolka** [comment]
Dear Violeta,

since I'm rather new to the SDC Tools User Group, I don't know anything about the SDC User Meeting. Could you provide some general information about it's contents and target audience?

Many thanks in advance and best regards,
Matthias

**Comment by violetacln** [comment]
Hi!
The support team organizes a meeting with users of SDC Tools each year. 
The agenda for this years' meeting is not finalized but the main content will be about:  recent developments from the software developers, their plans for the future and, most importantly answering users' questions / discuss topics proposed by them. Users may give presentations as well, about their experience or interesting problems encountered while employing the sdc Tools. 
Feel free to propose (send me an email for instance) such topics or short presentations! 
Target audience consists of experts from Statistical Institutes but not only: any other Institutes which have to solve data disclosure control problems.
